nbbuild/cluster.properties

```diff
@@ -220,6 +220,7 @@ nb.cluster.platform=\
         libs.jna.platform,\
         libs.jsr223,\
         libs.junit4,\
+        libs.mockito,\
```

Comment:
Really? Another 3rd party library into the platform? Is there some good justification?